docs/docsource/reference/introduction/index.rst

```diff
@@ -1,29 +1,29 @@
 Introduction
 ============
 
-Zenko, Scality's multi-cloud controller, provides an open-source,
+XDM, Scality's multi-cloud controller, provides an open-source,
 platform-agnostic gateway to facilitate data replication and management for
-storage managers handling extreme data volumes over multiple clouds. Zenko
+storage managers handling extreme data volumes over multiple clouds. XDM  
```

Comment:
Seeing the extra spaces a bit here, maybe a typo in sed or something?

Reply:
an on-purpose kludge.